students/myungkookyoon/users/views.py

```diff
@@ -48,13 +49,17 @@ def post(self, request):
             email    = data['email']
             password = data['password']
             
-            if not User.objects.filter(email = email).exists():
-                return JsonResponse({"message": "INVALID_USER"}, status = 401)
+            if not User.objects.filter(email=email).exists():
+                return JsonResponse({'MESSAGE':'INVALID_USER'}, status=401)
```

Comment:
```suggestion
                return JsonResponse({'message':'INVALID_USER'}, status=401)
```
보통 키값은 소문자로 많이 사용합니다.

Reply:
네 확인했습니다 수정하겠습니다!